tests/ci/profile_handler.go

```diff
@@ -424,7 +422,7 @@ func DestroyRHCSClusterByProfile(token string, profile *Profile) error {
 	clusterService, err := EXE.NewClusterService(profile.ManifestsDIR)
 	Expect(err).ToNot(HaveOccurred())
 	clusterArgs := &EXE.ClusterCreationArgs{
-		Token:              token,
+		// Token:              token,
```

Comment:
why commented and not removed?

Reply:
eagle eyes.  I replaced with a regex. Seems this one is left.